.github/workflows/tests-dbstorage.yml

```diff
@@ -1,4 +1,4 @@
-name: Tests (RDB storage)
+name: Tests (DB storage)
```

Comment:
Could you update the job name of `test-rdbstorage` in L16 as well? I guess `test-storage-with-server` might explain the contents of the job.

Reply:
I already mentioned about it at the first commit log, "I left the job name as `tests-rdbstorage` because I do not have permission to edit `Branch protection rules`". When we change the job name, we should also change the `Branch protection rules`. Without it, CI will fail.

So could you please modify it after this PR is merged?

Reply:
Thank you for your explanation. I overlooked the protection.
In my understanding, a CI file which does not exist in the `optuna/optuna` master will be just ignored. Could you share the error massages of the CI failure if you still have it?

> the first commit log

I'm sorry but I missed [it](https://github.com/optuna/optuna/pull/3258/commits/0748e651c45e7a0fbc2d45c266e48fd4a6c2de5a). If you put it in the description of the PR, the reviewers as well as future developers can  easily access the reason why you keep the file name.

Reply:
> Could you share the error massages of the CI failure if you still have it?

I made a PR #3300 that shows effect of renaming the job name.
RDB Storage tests itself already finished but tests-rdbstorage (3.x) still waits for finish of old name jobs.

Reply:
> RDB Storage tests itself already finished but tests-rdbstorage (3.x) still waits for finish of old name jobs.

I see. `test-rdbstorage` is makred as required, and the corresponding jobs could not be found after renaming the job name. Thank you for your explanation.

So, it's impossible to rename it with this PR. Then, could you add a TODO comment to rename the job name?

Reply:
> could you add a TODO comment to rename the job name?

Fixed.

Reply:
Thank you!